frontend/src/components/VCSProviderBasicInfoPanel.vue

```diff
@@ -163,7 +195,27 @@ export default {
       }
     };
 
-    return { state, namePlaceholder, instanceUrlLabel, changeUrl };
+    // FIXME: Unexpected mutation of "config" prop. Do we care?
```

Comment:
Is this a problem? There are plenty of existing places are doing the same.

Reply:
@LiuJi-Jim